test/recipes/30-test_evp_data/evpmac.txt

```diff
@@ -157,7 +157,7 @@ Output = 5150d1772f50834a503e069a973fbd7c
 MAC = SipHash
 Ctrl = digestsize:13
 Key = 000102030405060708090A0B0C0D0E0F
-Result = MAC_CTRL_ERROR
+Result = MAC_BAD_PARAMS
```

Comment:
Is this a breaking change?

Reply:
No.  "MAC_BAD_PARAMS" is a string generated by `test/evp_test.c`